### ISSUE TITLE
"ImportError: No module named disttest"

### ISSUE THREAD

**Issue by mgaitan** [jaredly/django-feedback#22]
```
(preciosa)tin@morochita:~/lab/preciosa$ pip install -U django-feedback
Downloading/unpacking django-feedback from https://pypi.python.org/packages/source/d/django-feedback/django-feedback-1.0.0.tar.gz#md5=65412dc9656cdec353befc8c5a61ee11
  Downloading django-feedback-1.0.0.tar.gz
  Running setup.py (path:/tmp/pip-build-rIhwwk/django-feedback/setup.py) egg_info for package django-feedback
    Traceback (most recent call last):
      File "<string>", line 17, in <module>
      File "/tmp/pip-build-rIhwwk/django-feedback/setup.py", line 6, in <module>
        from disttest import test
    ImportError: No module named disttest
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 17, in <module>

  File "/tmp/pip-build-rIhwwk/django-feedback/setup.py", line 6, in <module>

    from disttest import test

ImportError: No module named disttest

----------------------------------------
Cleaning up...
Command python setup.py egg_info failed with error code 1 in /tmp/pip-build-rIhwwk/django-feedback
Storing debug log for failure in /home/tin/.pip/pip.log
```

**Comment by PetrDlouhy** [comment]
It seems to be Python 3 compatibility issue. Are you running this on Python 3?

**Comment by mgaitan** [comment]
yes I am, isn't it compatible? what a pity, it's 2016!
https://twitter.com/jakevdp/status/645288857382944768

**Comment by PetrDlouhy** [comment]
It should be compatible, but evidently something left unnoticed. But you should provide as much information as possible, if you want to make the chance of fixing the bug higher.

**Comment by kinkerl** [comment]
I have the same error

ubuntu 15.10, python 2.7.10. clean new virtualenv without site packages.

**Comment by PetrDlouhy** [comment]
I have added the fix for that. It is not tested at all. Can you please try installing from git?

**Comment by mgaitan** [comment]
```
(preciosa)tin@morochita:~/lab/preciosa$ pip install -U https://github.com/jaredly/django-feedback/archive/master.zip
Downloading/unpacking https://github.com/jaredly/django-feedback/archive/master.zip
  Downloading master.zip (47kB): 47kB downloaded
  Running setup.py (path:/tmp/pip-oyeND2-build/setup.py) egg_info for package from https://github.com/jaredly/django-feedback/archive/master.zip
    Traceback (most recent call last):
      File "<string>", line 17, in <module>
      File "/tmp/pip-oyeND2-build/setup.py", line 6, in <module>
        from .disttest import test
    ValueError: Attempted relative import in non-package
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 17, in <module>

  File "/tmp/pip-oyeND2-build/setup.py", line 6, in <module>

    from .disttest import test

ValueError: Attempted relative import in non-package

----------------------------------------
Cleaning up...
Command python setup.py egg_info failed with error code 1 in /tmp/pip-oyeND2-build
Storing debug log for failure in /home/tin/.pip/pip.log
(preciosa)tin@morochita:~/lab/preciosa$ python --version
Python 2.7.10

```

**Comment by Rewerson** [comment]
Got the same result as mgaitan with this fix on CentOS 6, Python 2.7.8.

**Comment by PetrDlouhy** [comment]
The master should be now OK. Please test.
